### PR TITLE
Refactor to use zest.releaser and tox

### PR DESCRIPTION
**Pull request by fbochu**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -1,32 +1,14 @@
...
-            raise CalledProcessError(retcode, cmd, output=output)
-        return output
 
+import pkg_resources
 import sphinx_rtd_theme
 
+
 project = u'zbarlight'
-copyright = u'2015, Polyconseil'
+copyright = u'2014, Polyconseil'
```
Comment (bgelineau) [inline docs/conf.py]: tu remontes le temps ?

File: docs/conf.py [hunk 2]
```diff
@@ -1,32 +1,14 @@
...
 
-version = check_output(
-    ['python', 'setup.py', '--version'],
-    cwd=os.path.join(os.path.dirname(__file__), '..'),
-).decode()
+version = pkg_resources.get_distribution('zbarlight').version
```
Comment (bgelineau) [inline docs/conf.py]: propre :)

File: docs/conf.py (hunk 1 above)
Comment (fbochu) [inline docs/conf.py]: The first zbarlight release was made on 15 Dec 2014

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py26, py27, py34, py35, docs, quality
+skip_missing_interpreters = True
+
+[testenv:docs]
+deps = -rrequirements-dev.txt
```
Comment (bgelineau) [inline tox.ini]: r en trop

**Pull request merged**